pandas/tests/extension/arrow/bool.py

```diff
@@ -76,6 +76,14 @@ def __getitem__(self, item):
     def __len__(self):
         return len(self._data)
 
+    def astype(self, dtype, copy=True):
```

Comment:
They wouldn't be hit without without the change to `internals/construction.py`, which would be hard to motivate without DatetimeArray, since it's unique in that `.dtype` is just maybe an ExtensionDtype.